x-pack/plugins/maps/public/classes/layers/solution_layers/security/security_layer_wizard.tsx

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { LayerWizard, RenderWizardArguments } from '../../layer_wizard_registry';
+import { getSecurityIndexPatterns } from './security_index_pattern_utils';
+import { SecurityLayerTemplate } from './security_layer_template';
+
+export const SecurityLayerWizardConfig: LayerWizard = {
+  checkVisibility: async () => {
+    const indexPatterns = await getSecurityIndexPatterns();
+    return indexPatterns.length > 0;
+  },
+  description: i18n.translate('xpack.maps.security.desc', {
+    defaultMessage: 'SIEM layers',
```

Comment:
Most all occurrences of `SIEM` have been replaced with either `Elastic Security` or `Security Solution`, so not sure if we still want to use SIEM here, or perhaps something like `Security layers` or `Network Security layers`?